modules/internal/ChapelRange.chpl

```diff
@@ -1747,10 +1977,12 @@ module ChapelRange {
   
     return (U(3), U(1));
   }
-  
+
+  pragma "no doc"
   inline proc chpl__extendedEuclid(u:int(32), v:int(32))
   { return chpl__extendedEuclidHelper(u,v); }
-  
+
+  pragma "no doc"
```

Comment:
Note that it also removes any functions that don't start with a letter so you can also remove the `no docs` from the overloaded methods.